### ISSUE TITLE
Why not set up a verification set（dev）

### ISSUE THREAD

**Issue by hhhh230** [crux82/ganbert#19]
I only saw the training set and the test set in your data，looking forward to your reply

**Comment by crux82** [comment]
The provided code is an example to show how to train and evaluate GAN-BERT. It is not meant to show how to do parameter optimization. In any case, I would suggest splitting the provided train dataset to obtain the development material. 

Hope it helps.
